Logger.Tests/FileLoggerTests.cs

```diff
@@ -1,10 +1,36 @@
 ﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
 
 namespace Logger.Tests
 {
     [TestClass]
     public class FileLoggerTests
     {
-        
+
+
+        [TestMethod]
+        public void FileLogger_Makes_Successful_Logs()
+        {
+            //Arrange
+            string testPath = Path.GetRandomFileName();
+
+            LogFactory logFactory = new LogFactory();
+            logFactory.ConfigureFileLogger(testPath);
+
+            var logger = logFactory.CreateLogger(nameof(FileLoggerTests));

```

Comment:
```suggestion
            FileLogger logger = new FileLogger(testPath);
```
Since we are testing the `FileLogger` class and not the `LogFactory` class you may want to directly instantiate a new `FileLogger` to avoid failing due to an issue with the `LogFactory` class. Typically in unit tests you want to only test one system at a time, or rather ,one unit of the code at a time. Any failures that may be encountered by the `LogFactory` should be found in unit tests specifically for the `LogFactory`.